src/wizard/node/awslambda.md

```diff
@@ -0,0 +1,48 @@
+---
+name: AWS Lambda
+doc_link: https://docs.sentry.io/platforms/node/guides/aws-lambda/
+support_level: production
+type: framework
+---
+Create a deployment package on your local machine and install the required dependencies in the deployment package. For more information, see [Building an AWS Lambda deployment package for Node.js](https://aws.amazon.com/premiumsupport/knowledge-center/lambda-deployment-package-nodejs/).
```

Comment:
> Create a deployment package
> in the deployment package

This duplication sounds odd

Reply:
Acknowledged. Not sure if there's a way to remove duplication in new way of wizard files. They are standalone files now. Merging for now.